src/subcommands/deploy.ts

```diff
@@ -98,13 +114,15 @@ export default async function (rawArgs: Record<string, any>): Promise<void> {
     include: args.include?.map((pattern) => normalize(pattern)),
     exclude: args.exclude?.map((pattern) => normalize(pattern)),
     dryRun: args.dryRun,
+    config: args.config,
+    saveConfig: args.saveConfig,
   };
 
   await deploy(opts);
 }
 
 interface DeployOpts {
-  entrypoint: URL;
+  entrypoint: string;
```

Comment:
why is this change requried?

Reply:
`deploy()` function updates the config file if the deployment ends successfully. The `opts: DeployOpts` parameter is what is passed to `configFile.maybeWrite()`. For this reason, we want to pass around the raw entrypoint instead of the file URL resulting from parsing it.